config/space-presets.json

```diff
@@ -48,7 +48,7 @@
     }
   },
   {
-    "id": "5apps",
+    "id": "xmpp-5apps",
```

Comment:
This was actually intentional, because the idea is that you can change the type of the same space when migrating, without moving everything.